pkg/auto/history/config/root.go

```diff
@@ -46,6 +54,14 @@ func (f *FieldMask) MarshalJSON() ([]byte, error) {
 	return protojson.Marshal((*fieldmaskpb.FieldMask)(f))
 }
 
+type Sampling struct {
+	// Schedule is the time schedule between samples being recorded.
+	// The default is every 1 hour at minute 0 < 0 * * * * >.
+	Schedule *jsontypes.Schedule `json:"schedule,omitempty"`
+	// Disabled is the flag used to disable sampling.
+	Disabled bool `json:"disabled,omitempty"`
+}
```

Comment:
This is a fairly big breaking change for all deployed sc-bos systems. You're changing the default behaviour from Pull to Schedule(1h) which could have large effects on a lot of different apps. The Ops UI, for one example, relies on occupancy being more frequent than 20mins, which this would break. I don't see a reason to change the default here.

If the schedule is no longer the default then you can get rid of the Disabled property, and the Sampling struct entirely. Enabling a schedule based collection can be turned on with the presence of the schedule in the json config. ParseConfig can also go away.